docs/reST/ref/code_examples/draw_module_example.py

```diff
@@ -1,94 +1,92 @@
-# Import a library of functions called 'pygame'
 import pygame

```

Comment:
if you want more readability and a bit more clean code then there are too many 'magic constants'. Code should be self explanatory without having to have to look up the function signatures. But is fine with me except for the arc call, see comments there.

Resolve at will.